packages/core/lib/config.js

```diff
@@ -113,9 +110,7 @@ const loadPreset = (context, preset) => {
   } catch (error) {
     if (!isResolveError(error)) throw error;
     try {
-      return loadConfig(
-        dirname(resolvePreset(context, join(preset, 'package.json')))
-      );
+      return loadConfig(dirname(resolve(context, `${preset}/package.json`)));
```

Comment:
Please use `path.join()` instead of hardcoding a `/`

Reply:
Since this is essentially a module path, I think it is more sensible to go for a `/` in this case.